server.js

```diff
@@ -114,12 +114,29 @@ let sendJavascript = (req, res) => {
     });
 };
 
+let postStaples = (req, res) => {
+    readBody(req, (body) => {
+        let stapleIngredients = JSON.parse(body);
```

Comment:
Do you think we would be able to pass the payload in from the checkTokens function? That way you wouldn't have to go through the process of getting it again in here

Reply:
After discussing this with you in person, it looks like we should leave the function as is for now!